src/ottplan.java

```diff
@@ -0,0 +1,141 @@
+import java.text.DecimalFormat;
+import java.util.*;
+abstract class Subscription{
+    protected String resolution;
+    protected String planName;
+    protected int noOfViewers;
+    protected double amount;
+    protected String dis;
+    abstract void calculate(boolean a);
+    void display()
+    {
+        DecimalFormat df=new DecimalFormat("##.00");
+        System.out.println(dis);
+        System.out.println("Plane Name:"+planName);
+        System.out.println("No. Of Concurrent Viewers:"+noOfViewers);
+        System.out.println("Supported Resolution:"+resolution);
+        System.out.println("Total Amount:"+df.format(amount));
+    }
+}
```

Comment:
Use `printf` for formatting output. 
Also, you could override the default `toString()` method and put this logic inside that